### DISCUSSION TITLE
Remove ExplorerPatcher

### DISCUSSION THREAD

**Discussion by simba83** [valinet/ExplorerPatcher#845] [Q&A]
Hello, Is there a way to uninstall ExplorerPatcher got some issues lately and would like to remove it.
But i cant find a easy way to do so.

**Comment by simba83** [comment]
Found a way around this here: https://github.com/valinet/ExplorerPatcher/discussions/287#discussioncomment-1648873

**Reply by valinet** [reply]
No, the answer to your question is actually [here](https://github.com/valinet/ExplorerPatcher#uninstalling).
